Sources/CustomerCenter/CustomerCenterConfigData.swift

```diff
@@ -550,6 +555,7 @@ extension CustomerCenterConfigData.Support {
 
     init(from response: CustomerCenterConfigResponse.Support) {
         self.email = response.email
+        self.shouldWarnCustomerToUpdate = response.shouldWarnCustomerToUpdate ?? false
```

Comment:
Defaults to false if it isn't included in the response from the backend since the checkbox's default value in the dashboard is false.